packages/sync/src/cache/CacheUpdates.ts

```diff
@@ -12,8 +13,19 @@ import { MutationUpdaterFn } from "apollo-client";
  *  createTask: () => {...}
  * }
  */
-interface CacheUpdates {
+export interface CacheUpdates {
   [key: string]: MutationUpdaterFn;
 }
 
-export default CacheUpdates;
+export type UpdateFunction = (array: [CacheItem], newItem?: CacheItem) => CacheItem[];
```

Comment:
This seems to be defined in Apollo. We introduce here a lot of types that may be available in Apollo. I'm more curious why this is happening? Are they private/not exposed?

Reply:
Hmm. I saw the mutation updater function, but this is different. i didn't see anything that matched this type - i'll take another look

Reply:
Ignore that!